src/Security/src/DataProtection.Redis/Steeltoe.Security.DataProtection.Redis.csproj

```diff
@@ -10,6 +10,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.DataProtection.StackExchangeRedis" Version="$(MatchTargetFrameworkVersion)" />
+    <PackageReference Include="Microsoft.Extensions.Caching.StackExchangeRedis" Version="$(MatchTargetFrameworkVersion)" />
```

Comment:
Is there any scenario where this reference wouldn't be wanted? I suspect it would generally already be present in applications using Redis, right?

Reply:
The reference was already there, but I removed it several weeks ago in the process of cleaning up unused references.

Reply:
The ref doesn't need to be in the app. For example if only using `IConnectionMultiplexer` without `IDistributedCache`.

Reply:
Oh, if we've previously shipped with that dependency then it isn't an issue

Reply:
I was wrong, another package was removed. So this one is new. Do you think that's problematic? We need it to configure `RedisCacheOptions`.

Reply:
The old connectors would do that via reflection I guess.

Reply:
Discussed offline, no issue.